index.js

```diff
@@ -37,15 +37,22 @@ module.exports = {
     };
 
     await appBlueprint.install(appOptions);
-    await Promise.all([
+
+    let tasks = [
       this.updateTestAppPackageJson(path.join(testAppPath, 'package.json')),
       this.overrideTestAppFiles(
         testAppPath,
         path.join(options.target, 'test-app-overrides')
       ),
       fs.unlink(path.join(testAppPath, '.travis.yml')),
-      this.setupReleaseIt(options.target),
-    ]);
+    ];
+
+    if (options.releaseIt) {
```

Comment:
1. Why are you touching tis in this PR, seems unrelated?
2. where does this option come from? AFAIU they are set by the `ember addon` command, so out of our control?

Reply:
> it errored for me locally, so I hid it behind a flag

Well, if there is an error, we should fix it, or at least create an issue? 
There was #11, but this should be fixed. 🤔 

> can we not add flags per blueprint implementation?

Idk, can we? Have you tested that this works, i.e. that you can opt-in? 

This is not running as part of `ember generate`, but `ember addon`, and I believe that this is where the options are defined for this command: https://github.com/ember-cli/ember-cli/blob/master/lib/commands/addon.js#L9-L29

Btw, if we are able to make this opt-in, then I think that is very much desirable, see #9! 

But again, I think this should be a separate PR!

Reply:
oh hey, this is relevant -- and you already linked it :sweat_smile:

Reply:
Reported: https://github.com/embroider-build/addon-blueprint/issues/21

Reply:
Extracted --release-it changes